src/ui/rootfs/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM bitnami/nginx
+
+COPY dist/ /app
```

Comment:
can you add a comment with the process that needs to be previously done?

`ng build --prod ....`